bedevere/bpo.py

```diff
@@ -37,6 +40,19 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = textwrap.dedent(f"""\
```

Comment:
Is this a bug with textwrap.dedent() or f-strings?
Worth opening an issue in b.p.o about this?